source/deployment/resyncing-conda.rst

```diff
@@ -8,29 +8,35 @@ operating cycle, to help isolate users from changes that might break things
 mid-cycle due to some unintended upgrade.
 
 Before the start of each cycle, we re-synchronize our internal collection of
-packages with the latest packages available from Anaconda.
+packages with the latest packages available from Anaconda (``defaults`` channel).
 
 The internal Anaconda server runs on the ``alexandria`` host, inside the Controls
-network, and it runs as a service account user ``anaconda-server``.
+network, and it runs as root.
 
 .. code-block:: bash
 
     ssh alexandria
-    sudo su anaconda-server
-    cd
-    export https_proxy=http://proxy:8888
-    export HTTPS_PROXY=http://proxy:8888
+    sudo su
+    bash /opt/builds/bootstrap-mirror-nsls2anaconda-local-folder
+    bash /opt/builds/run_conda_index
 
-The configuration file ``~/anaconda.yaml`` controls which packages are synced.
-Edit it, for example, to update which version(s) of Python we want packages for
-this cycle.
+``bootstrap-mirror-nsls2anaconda-local-folder`` script can be easily written
+based on `bootstrap-tag-build <https://github.com/NSLS-II/lightsource2-recipes/blob/master/scripts/bootstrapping/bootstrap-tag-build>` and
+`nsls2-tag-build-local <https://github.com/NSLS-II/lightsource2-recipes/blob/master/scripts/nsls2-tag-build-local.sh>`_.
+Token needs to be provided in the script. ``conda index`` needs to run every time
+new package is added. Information about ``conda index`` can also be found at
+`create-custom-channels <https://conda.io/docs/user-guide/tasks/create-custom-channels.html>`_.
 
-Finally, the command to sync is:
+Moreover, sync of ``lightsouce2-tag`` to ``nsls2-tag`` is similar:
```

Comment:
I think this should be written as `Moreover a sync of ``lightsource2-tag`` to ``nsls2-tag`` follows a similar process:`